### PR TITLE
PMM-1458 Create reference document

### LINKED ISSUE #477: v0.31.2 MongoDB Exporter - centos 7 -  does not gather any metric
**Describe the bug**
http://fqdn:9130/metrics --> does not show any mongodb metric. It just shows the default go lang metrics and mongodb_up gauge.

**To Reproduce**
1. system - CentOS Linux release 7.9.2009 (Core)  , mongodb version - ``4.4.13``
2. mongodb_exporter version - Version: `v0.31.2` Commit: 5ceed65
3. User Permissions
``{
    user: "username",
    pwd: "pwd",
    roles: [
        { role: "clusterMonitor", db: "admin" },
        { role: "read", db: "local" }
    ]
}``
5. Exporter is deployed as a systemd service file.
 ```
[Unit]
Description=MongoDB Exporter
After=network.target
Documentation=https://github.com/percona/mongodb_exporter
[Service]
User=mongod
Group=mongod
EnvironmentFile=/etc/mongodb_exporter/mongodb_exporter
ExecStart=/usr/local/bin/mongodb_exporter --web.listen-address=fqdn:9130
ExecReload=/bin/kill -HUP $MAINPID
ExecStop=/bin/kill -15 $MAINPID
KillMode=process

[Install]
WantedBy=default.target
```
cat /etc/mongodb_exporter/mongodb_exporter
`MONGODB_URI=mongodb://user:pwd@fqdn:27017/admin`

**Expected behavior**
``curl http://fqdn:9130/metrics`` usually shows all the metrics. Now I see only default golang metrics and mongodb_up gauge.

**Logs**
The are no connectivity errors or issues reported in systemd logs or mongod logs. The exporter is able to connect to the DB. mongodb_up gauge is visible but no other metric beyond that is produced.

**Additional context**
The issue does not seem to occur with [v0.20.9](https://github.com/percona/mongodb_exporter/releases/tag/v0.20.9)
Any release beyond that is not working

### PR DESCRIPTION
**Pull request by ShashankSinha252**
[PMM-1458](https://jira.percona.com/browse/PMM-1458) : Update documentation for MongoDB exporter

Resolves #477

### CODE REVIEW

**Review by denisok — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -135,6 +114,9 @@ The labels are:
 - rs_state: Replicaset state is an integer from `getDiagnosticData()` -> `replSetGetStatus.myState`. 
 Check [the official documentation](https://docs.mongodb.com/manual/reference/replica-states/) for details on replicaset status values.
 
-## Submitting Bug Reports and adding new functionality
+## Usage Reference
+
+Check out [Reference Guide](REFERENCE.md) for details on using exporter.
```
Comment (rasika-chivate) [inline README.md:119]:
```suggestion
See the [Reference Guide](REFERENCE.md) for details on using the exporter.
```

File: README.md [hunk 2]
```diff
@@ -135,6 +114,9 @@ The labels are:
...
+## Bug Reports / Feature PR
 
-please see [Contribution Guide](CONTRIBUTING.md)
+Please read [Contribution Guide](CONTRIBUTING.md).
```
Comment (rasika-chivate) [inline README.md:122]:
```suggestion
 Refer to the [Contribution Guide](CONTRIBUTING.md).
```

**Review by rasika-chivate — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ShashankSinha252) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (ShashankSinha252) [reply README.md]: Done

**Comment by ShashankSinha252** [comment]
@rasika-chivate please review PR again.

**Review by rasika-chivate — APPROVED** [review]

**Pull request merged**